jsonxx.cc

```diff
@@ -307,6 +307,7 @@ bool Object::parse(std::istream& input, Object& object) {
             delete v;
             break;
         }
+        JSONXX_ASSERT(object.value_map_.find(key) == object.value_map_.end());
```

Comment:
Since an invalid input would cause the check to fail. You should just return `false` here instead of asserting.